include/os/freebsd/spl/sys/uio.h

```diff
@@ -39,31 +39,43 @@
 
 #define	uio_loffset	uio_offset
 
-typedef	struct uio	uio_t;
 typedef	struct iovec	iovec_t;
 typedef	enum uio_seg	uio_seg_t;
 
+typedef struct zfs_uio {
+	struct uio	*uio;
+} uio_t;
+
+#define	GET_UIO_STRUCT(u)	(u)->uio
+#define	uio_segflg(u)		GET_UIO_STRUCT((u))->uio_segflg
```

Comment:
```suggestion
#define	uio_segflg(u)		GET_UIO_STRUCT(u)->uio_segflg
```

I may be mistaken, but I don't think the double parenthesis are needed here since `(u)` is used in `GET_UIO_STRUCT`.

Reply:
I removed the double parens.